app/controllers/messages_controller.rb

```diff
@@ -215,7 +215,6 @@ def get_time_from_string(string)
       else
         time = Chronic.parse(string, :context => :past)
       end
-    rescue
```

Comment:
This can't be blindly deleted. I agree that `Exception` should not be blindly rescued like this, but it can't be removed without thought to why it's there.